app.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controllers");
+
+const app = express();
+app.use(express.json());
```

Comment:
This is not needed as the server doesn't deal with the incoming request body as of yet.